libs/gi/ui/src/components/artifact/editor/index.tsx

```diff
@@ -611,7 +611,13 @@ export function ArtifactEditor({
                 setLocation={(location) => update({ location })}
                 autoCompleteProps={{ disabled: !artifact }}
               />
-
+              <CardThemed bgt="light" sx={{ py: 1, px: 2 }}>
+                <Grid container spacing={1}>
+                  <Grid item flexGrow={1}>
+                    Total Rolls: {cArtifact?.totalRolls}
```

Comment:
If its undefined, maybe make it say `{guessedTotalRolls} (ambiguous)` or something

Reply:
No we can just use the total rolls of our guessed artifact. But don't assert it as a fact if it's ambiguous
E.g. if the roll guessing function returns 8 rolls but it could be 9 rolls, display the 8 with some ux that shows it's ambiguous, and don't store the 8 in totalrolls